### PR TITLE
Feature/storage/stg73comments

### LINKED ISSUE #13187: [Storage] Address comments in STG 73 PR
https://github.com/Azure/azure-sdk-for-net/pull/13105

### PR DESCRIPTION
**Pull request by seanmcc-msft**
Resolves https://github.com/Azure/azure-sdk-for-net/issues/13187

STG 73 PR - https://github.com/Azure/azure-sdk-for-net/pull/13105

### CODE REVIEW

**Comment by seanmcc-msft** [comment]
@tg-msft

File: sdk/storage/Azure.Storage.Blobs.ChangeFeed/src/ChangeFeedFactory.cs [hunk 1]
```diff
@@ -39,10 +40,11 @@ internal class ChangeFeedFactory
         }
 
         public async Task<ChangeFeed> BuildChangeFeed(
```
Comment (jaschrep-msft) [inline sdk/storage/Azure.Storage.Blobs.ChangeFeed/src/ChangeFeedFactory.cs:42]: `BuildChangeFeedInternal`

File: sdk/storage/Azure.Storage.Blobs.ChangeFeed/src/ChangeFeedFactory.cs [hunk 2]
```diff
@@ -170,15 +178,17 @@ internal class ChangeFeedFactory
         }
 
         internal async Task<Queue<string>> GetYearPaths(
```
Comment (jaschrep-msft) [inline sdk/storage/Azure.Storage.Blobs.ChangeFeed/src/ChangeFeedFactory.cs:180]: `GetYearPathsInternal`

**Review by jaschrep-msft — APPROVED** [review]
Good changes. Couple names to update with our patterns.

File: sdk/storage/Azure.Storage.Blobs.ChangeFeed/src/ChangeFeedFactory.cs (hunk 1 above)
Comment (jaschrep-msft) [reply sdk/storage/Azure.Storage.Blobs.ChangeFeed/src/ChangeFeedFactory.cs:42]: Looks public in this, but is on an internal class.

Comment (seanmcc-msft) [reply sdk/storage/Azure.Storage.Blobs.ChangeFeed/src/ChangeFeedFactory.cs:42]: I think we should be fine - [Public things on an internal class are still internal](https://stackoverflow.com/questions/9302236/why-use-a-public-method-in-an-internal-class).

**Review by kasobol-msft — APPROVED** [review]

File: sdk/storage/Azure.Storage.Blobs.ChangeFeed/src/ChangeFeedFactory.cs (hunk 2 above)
Comment (seanmcc-msft) [reply sdk/storage/Azure.Storage.Blobs.ChangeFeed/src/ChangeFeedFactory.cs]: Fixed in next iteration.

**Pull request merged**